### PR TITLE
Single Fetch

### PR DESCRIPTION
**Pull request by brookslybrand**
Here's hoping this makes Fastly graphs go 📈

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+force=true
```
Comment (brookslybrand) [inline .npmrc:1]: Don't want to leave this in long term, but makes it easier to deal with experimental releases here

File: app/entry.server.tsx [hunk 2]
```diff
@@ -1,23 +1,152 @@
-import type { HandleDocumentRequestFunction } from "@remix-run/node";
+import { PassThrough } from "node:stream";
+
+import type { AppLoadContext, EntryContext } from "@remix-run/node";
+import { createReadableStreamFromReadable } from "@remix-run/node";
 import { RemixServer } from "@remix-run/react";
-import { renderToString } from "react-dom/server";
```
Comment (brookslybrand) [inline app/entry.server.tsx:3]: Single fetch does not work with `renderToString` as @brophdawg11 pointed out. Went ahead and deleted these files and ran `npx remix reveal`. Let me know if there's any issue with that approach

**Review by brophdawg11 — CHANGES_REQUESTED** [review]
Just dropping this here to block merging until I can get a final experimental cut - Jacob and I made a few final tweaks since the version this is pinned to

File: app/entry.server.tsx [hunk 3]
```diff
@@ -1,23 +1,152 @@
...
+  request: Request,
+  responseStatusCode: number,
+  responseHeaders: Headers,
+  remixContext: EntryContext
+) {
+  return new Promise((resolve, reject) => {
+    let shellRendered = false;
+    const { pipe, abort } = renderToPipeableStream(
+      <RemixServer
+        context={remixContext}
```
[the remark this answers is not in the document]
Comment (jacob-ebey) [reply app/entry.server.tsx:114]: We missed a type that's "shared" between runtimes somewhere. Package re-org can't come soon enough.

[the remark this answers is not in the document]
Comment (brophdawg11) [reply app/entry.server.tsx:114]: huh - yeah I'll see what the error is - I thought we got the right fields added on both spots

File: app/entry.server.tsx (hunk 2 above)
Comment (brophdawg11) [reply app/entry.server.tsx:3]: Want to just keep them deleted if we don't do anything custom in them?

**Comment by brophdawg11** [comment]
Updated to the latest experimental off `dev` in [a6325ce](https://github.com/remix-run/remix-website/pull/222/commits/a6325cef9bea7151afa2a10be31a7cd7699e1835)

File: app/entry.server.tsx (hunk 2 above)
Comment (brophdawg11) [reply app/entry.server.tsx]: I deleted them in my update - but feel free to add them back if you want them in!

File: app/entry.server.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (brophdawg11) [reply app/entry.server.tsx]: Fixed - was the `renderMeta.streamCache`

**Review by brophdawg11 — APPROVED** [review]

**Pull request merged**